tools/azure-sdk-tools/devtools_testutils/proxy_startup.py

```diff
@@ -87,6 +87,17 @@ def check_availability() -> None:
         return 404
 
 
+def check_system_proxy_availability() -> None:
+    """Checks for SSL_CERT_DIR and REQUESTS_CA_BUNDLE environment variables."""
+    ssl_cert = "SSL_CERT_DIR"
+    ca_bundle = "REQUESTS_CA_BUNDLE"
+
+    if not os.environ.get(ssl_cert):
+        _LOGGER.error(f"Please ensure the '{ssl_cert}' environment variable is correctly set in your test environment")
+    if not os.environ.get(ca_bundle):
```

Comment:
After thinking about this again, I realized that we only need these environment variables set when we're using an SSL connection. That's true by default, but if someone sets [PROXY_URL](https://github.com/Azure/azure-sdk-for-python/blob/5fd52b9ee039f8711322bd7ea43af763d326291a/tools/azure-sdk-tools/devtools_testutils/config.py#L8) to be something like `"http://localhost:5000"`, we shouldn't raise error logs about certificate setup
```suggestion
    if PROXY_URL.startswith("https") and not os.environ.get(ssl_cert):
        _LOGGER.error(f"Please ensure the '{ssl_cert}' environment variable is correctly set in your test environment")
    if PROXY_URL.startswith("https") and not os.environ.get(ca_bundle):
```

Reply:
Thank you for your thorough review!